packages/beacon-node/src/network/reqresp/beaconBlocksMaybeBlobsByRoot.ts

```diff
@@ -21,7 +21,8 @@ import {
   getBlockInputBlobs,
   getBlockInputDataColumns,
 } from "../../chain/blocks/types.js";
-import {BlockInputAvailabilitySource} from "../../chain/seenCache/seenGossipBlockInput.js";
+import {ChainEventEmitter} from "../../chain/emitter.js";
+import {BlockInputAvailabilitySource, getDataColumnsFromExecution} from "../../chain/seenCache/seenGossipBlockInput.js";
```

Comment:
Noticed this also get imported from this file.  Lets move that with the other utils

Reply:
did you mean `BlockInputAvailabilitySource`, or just the `getDataColumnsFromExecution`?